### ISSUE TITLE
Snap to guide  

### ISSUE THREAD

**Issue by CoreFox** [cocos2d/CocosBuilder#196]
I'v been trying to use the snap to guid feature to create a tile map scene, but cant get any sprites to snap to the guides. 

So I was wondering whats the priority on the snap to guid feature and if it would be working on the next stable version. Was also wondering if there would be a way to automatically set up a grid with the guides or with dots (each dot acts as a snap point) and adjust its spacing.

Snap to guid would make tiled sprite alignment so simple.

**Comment by vlidholt** [comment]
I agree this would be a good feature! I'll try to add it at some point.
